tests/integration/models/test_cohort.py

```diff
@@ -59,14 +59,44 @@ def fixture_static_filter_parent_child(create_cohort_db, db_session, fixture_coh
         parent_id=parent_filter.id,
         cohort_id=fixture_cohort.id,
         filters=[{"field": "cases.primary_site", "value": ["bronchus and lung"]}],
-        static=True,
+        cohort_type="static",
```

Comment:
Consider pulling the hardcoded strings into an enum and then print the enum value as the string (assuming "static" and "dynamic" are used in the module under test)

Reply:
Good suggestion, I'm making the update